server/app/media_export/png_exporter.py

```diff
@@ -0,0 +1,101 @@
+import json
+import matplotlib.pyplot as plt
+import matplotlib.patches as patches
+from PIL import Image
+from io import BytesIO
+import base64
+import numpy as np
+import zipfile
+import os
+import io
+import sys
+
+sys.path.append(os.path.abspath(os.path.dirname(__file__)))
+
+_START_DEFAULT = 0
+_END_DEFAULT = 9999999
+
+def apply_tint(image, color):
+    r, g, b, a = np.rollaxis(image, axis=-1)
+    r = np.clip(np.uint8(r * color[0]), 0, 255)
+    g = np.clip(np.uint8(g * color[1]), 0, 255)
+    b = np.clip(np.uint8(b * color[2]), 0, 255)
+    tinted_image = np.stack([r, g, b, a], axis=-1)
+    return tinted_image
+
+def create_PNGs(vfg_file, startStep=_START_DEFAULT, stopStep=_END_DEFAULT):

```

Comment:
Given that the mp4, gif and png functions are quite similar, would it be better to refactor it and reuse code so it is easier to maintain?